cmd/podman/login.go

```diff
@@ -48,7 +51,11 @@ func init() {
 	completion.CompleteCommandFlags(loginCommand, auth.GetLoginFlagsCompletions())
 
 	// Podman flags.
-	flags.BoolVarP(&loginOptions.tlsVerify, "tls-verify", "", false, "Require HTTPS and verify certificates when contacting registries")
+	secretFlagName := "secret"
+	flags.BoolVar(&loginOptions.tlsVerify, "tls-verify", false, "Require HTTPS and verify certificates when contacting registries")
+	flags.String(secretFlagName, "", "Retrieve passwd from secret file")
```

Comment:
Should we be using `flags.StringVar` instead? Looks like all the other places in our code, `flags.String` is only used for noops